includes/wikia/VariablesBase.php

```diff
@@ -8964,3 +8964,17 @@
  * @var bool
  */
 $wgEnableQualtricsSiteInterceptExt = false;
+
+/**
+ * Enables Quizzes extension on select communities
+ * @see CAKE-4585
+ * @var bool
+ */
+$wgEnableTriviaQuizzesAlpha = false;
```

Comment:
Let's name this like `$wgEnableTriviaQuizzesExt` so that (1) we know it's the on/off switch for the extension and (2) it follows the naming convention of on/off variables for other MW extensions.

Reply:
Alright, that's fair enough.